### PR TITLE
Add rcvr hookup

### PR DESCRIPTION
**Pull request by david-deboer**
Renames corr_handling and implements PAM retrieval.

### CODE REVIEW

File: hera_mc/cm_hookup.py [hunk 1]
```diff
@@ -279,6 +279,32 @@ def get_correlator_input_from_hookup(self, hookup_dict):
                     corr_input[j] = p[-1].downstream_part
         return corr_input
 
+    def get_pam_from_hookup(self, hookup_dict, pam_name='post-amp'):
+        """
+        Retrieve the PAM connections from a hookup dictionary.
+        No rationalization is provided.
```
Comment (pkgw) [inline hera_mc/cm_hookup.py]: I don't understand this sentence.

File: hera_mc/cm_hookup.py [hunk 2]
```diff
@@ -279,6 +279,32 @@ def get_correlator_input_from_hookup(self, hookup_dict):
...
+            location example: RI4A1E:A
+            pam number example: RCVR93:A (for a PAPER RCVR)
+            pam number example: PAM75101:B (for a HERA PAM)
+        """
+        if len(hookup_dict['hookup'].keys()) > 1:
+            raise RuntimeError('Too many hookups provided to give e/n PAM inputs.')
+        pams = {}
+        # search through the connection path for the pam_name
+        pam_ind = 0
+        for item in hookup_dict['parts_epoch']['path']:
```
Comment (pkgw) [inline hera_mc/cm_hookup.py]: What data type is `hookup_dict['parts_epoch']['path']` ? If it's a list, could write

```python
pam_ind = hookup_dict['parts_epoch']['path'].index(pam_name)
```

File: hera_mc/cm_hookup.py (hunk 1 above)
Comment (david-deboer) [reply hera_mc/cm_hookup.py]: Me neither.

File: hera_mc/cm_hookup.py (hunk 2 above)
Comment (david-deboer) [reply hera_mc/cm_hookup.py]: Agreed.  I don't know if Adam had anything particular in mind but I'll change it.

File: hera_mc/cm_hookup.py (hunk 1 above)
Comment (david-deboer) [reply hera_mc/cm_hookup.py]: I see, they are having fun with the comment in the previous method.

**Pull request merged**